tests/component/_task_modules/test_triggers.py

```diff
@@ -139,64 +140,154 @@ def test___timestamp_not_enabled___wait_for_valid_timestamp___throw_error(
     assert exc_info.value.error_code == DAQmxErrors.TIMESTAMP_NOT_ENABLED
 
 
+@pytest.mark.parametrize(
+    "trig_src, trig_slope, trig_level",
+    [
+        ("APFI0", Slope.RISING, 2.0),
+        ("APFI0", Slope.FALLING, 3.0),
+        ("APFI1", Slope.FALLING, -2.0),
+        ("APFI1", Slope.RISING, -3.0),
+    ],
+)
 def test___start_trigger___cfg_anlg_edge_start_trig___no_errors(
     sim_6363_ai_voltage_task: Task,
+    trig_src: str,
+    trig_slope: Slope,
+    trig_level: float,
 ):
+    device_name = sim_6363_ai_voltage_task.devices[0].name
     sim_6363_ai_voltage_task.timing.cfg_samp_clk_timing(1000)
     sim_6363_ai_voltage_task.triggers.start_trigger.cfg_anlg_edge_start_trig(
-        trigger_source="APFI0", trigger_slope=Slope.RISING, trigger_level=2.0
+        trigger_source=trig_src, trigger_slope=trig_slope, trigger_level=trig_level
     )
 
     sim_6363_ai_voltage_task.start()
```

Comment:
You don't need this

Reply:
Noted, it's `.start()` is removed for new cases.